core/tests/protractor_desktop/preferences.js

```diff
@@ -64,6 +64,36 @@ describe('Preferences', function() {
     preferencesPage.expectUserBioToBe('Junior student from USA studying CS!');
   });
 
+  it('should change prefered audio language of the learber', function() {
```

Comment:
I think you mean "learner" :)

Reply:
Yeah, corrected as soon as I sent in the changes :P